lab/lab1/part4-project-euler.html

```diff
@@ -32,6 +32,11 @@
 
     ===================== */
 
+    var sum = 0;
+    for (let i = 1; i <= 1000; i++) {
+      if (i%3 === 0 || i%5 == 0) {sum += i;}
+    }
+    console.log(sum)
 
 
```

Comment:
Given that FizzBuzz was above and beyond... I suppose missing fibonacci is alright.